src/Sylius/Bundle/ApiBundle/Command/ContactMessageAwareInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/** @experimental */
+interface ContactMessageAwareInterface extends CommandAwareDataTransformerInterface
```

Comment:
Why did you add this interface?

Reply:
I was wondering where should I actually implement `getMessage()` method and it ended up here

Reply:
you can check how have we done it in the rest of the commands, this interface is redundant, our commands are public and we do not need interfaces for all methods